hack/deploy_loki_using_observatorium_yaml.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+LOKI_PROJECT_NAME=observatorium-logs-test
+LOKI_S3_SECRET_NAME=loki-objectstorage-secret
+
+delete_project_if_exists() {
+  PROJECT=$(oc get project | grep $1)
+  if [ -n "$PROJECT" ]; then
+    echo "Deleting $1 project/namespace"
+    oc delete project $1
+    while : ; do
+      PROJECT=$(oc get project | grep $1)
+      if [ -z "$PROJECT" ]; then break; fi
+      sleep 1
+    done
+  fi
+}
+
+create_project() {
+  oc new-project $1
+}
+
+get_observatorium_template() {
+  rm -f observatorium-logs-template.yaml
+  wget https://raw.githubusercontent.com/rhobs/configuration/main/resources/services/observatorium-logs-template.yaml 
+}
+
+create_objectstorage_s3_secret() {
+rm -f $LOKI_S3_SECRET_NAME.yaml
+cat > $LOKI_S3_SECRET_NAME.yaml << EOF
+apiVersion: v1
+kind: Secret
+stringData :
+  endpoint: https://s3.us-east-1.amazonaws.com
+  bucketnames: <s3-bucket-name>
+  bucket: <s3-bucket-name>
+  aws_region: us-east-1
+  aws_access_key_id: <aws-access-key>
+  aws_secret_access_key: <aws-secret-key>
```

Comment:
How do you envision to replace the placeholders with meaningful data?

Reply:
This is hacking code that I expect people to modify manually ... based on their specific environments to use ... the value is that this exposes the exact needed keys and values needed for the secret ... maybe in later versions of this we can add some environment variables and/or command line variables ... but I do not want to spend time on that for now

Reply:
Can we make these input variables which make the script to fail if nothing provided?

Reply:
In addition adding base64 encoding would be cool for users, e.g. `echo "$AWS_ACCESS_KEY_ID | base64`

Reply:
@periklis done ... added options to input using env variables and encoded into base64

Reply:
@periklis also added failures if parameters are not provided